adding full table (#19)
jhudsl/ITCR_Tables: platformsOverview.Rmd

---
title: "**Overview of the Platforms**"
output:
  html_document
---

<style type="text/css">
  body{
  font-size: 12pt;
  line-height: 1.6;
  text-align: justify;
  text-justify: inter-word;
}
</style>

<br>

```{r, fig.align='center', fig.alt="Banner", out.width="1540px", echo = FALSE}
knitr::include_graphics("resources/images/bannerITCR.png")
```

<br> 

### **Overview**

> This resources on this website aims to aid cancer researchers find bioinformatics computing platforms and data resources, many of which are funded by the ITCR (Informatics Technology for Cancer Research) program of the NCI (National Cancer Institute). Here is the glossary for relevant terms used in the website:

Reviewer: ```suggestion
This resources aims to improve access to bioinformatics computing platforms many of which are funded by the ITCR (Informatics Technology for Cancer Research) program of the NCI (National Cancer Institute). Here is the glossary for relevant terms used in these tables:
```

>   - **Platforms** are web or cloud-based websites/webpages that host computational tools or data resources.  

Reviewer: ```suggestion
  - **Platforms** are web or cloud-based websites/webpages that allow users to analyze their own data and/or publicly available data.
```

>   - **Multi-data type platforms** work with a wide variety of data types, from imaging, histological, omics, and clinical data. 

Reviewer: ```suggestion
  - **Multi-data type platforms**allow users to work with more than one type of data, including a wide variety of data types, from imaging, histological, omics, and clinical data. 
  ```

  
  - **Imaging** platforms work with clinical and pre-clinical images containing pathology imaging to radiographic imaging (e.g., CT, MRI, PET) obtained by specialized medical imaging devices.
  
  - **Omics** platforms work with proteomic, transcriptomic, genomic, metabolomic, lipidomic, and epigenomic data, which correspond to global analyses of data on proteins, RNA, genes, metabolites, lipids, and methylated DNA or modified histone proteins in chromosomes.  
  
  - **Clinical** platforms work with several types of clinical data, such as EMR Text and Clinical Notes, utilizing Natural Language Processing techniques, AI and machine learning algorithms.
  

>   - **Data resources** are remotely hosted sources of curated or collaborative bioinformatics data containing genomics, proteomics, imaging (histology and radiology), as well as other various types of data.

Reviewer: ```suggestion
  - **Data resources** are remotely hosted sources of curated or collaborative bioinformatics data containing genomics, proteomics, imaging (histology and radiology), as well as other various types of data. See the Data Resource page for more information.
  ```

>   - **Data resources** are remotely hosted sources of curated or collaborative bioinformatics data containing genomics, proteomics, imaging (histology and radiology), as well as other various types of data.

Reviewer: This reminds me that we might want to change the names for the RMD files so that the links are more informative. I can do that as it is a bit tricky, but we will want to link to that afterward.
